index.bs

```diff
@@ -270,19 +274,21 @@ the user agent <em>must</em> run the following steps:
 When <dfn>parsing an animator timeline list</dfn> for {{animatorCtor}},
 the user agent <em>must</em> run the following steps:
 
-    1. Let |properties| be an empty <code>sequence&lt;DOMString></code>.
+    1. Let |timelines| be an empty <code>sequence&lt;Object></code>.
 
     2. Let |timelinesIterable| be the result of <a>Get</a>(|animatorCtor|, |timelines|).
 
     3. For each |timeline| in |timelinesIterable|, run the following steps:
 
-        - If |timeline| is not "document" or "scroll", <a>throw</a> a <a>NotSupportedError</a>
-            and abort all these steps.
+        - If <a>Type</a>(|timeline|) is not {{DocumentTimelineOptions}} or {{ScrollTimelineOptions}}
```

Comment:
I think we need to add 'type' to the structure since in practice these will both be Object. But if we define 'type' in our dictionary then there's a risk of conflicting with the timeline options so perhaps it's safest to use {'type': 'document' || 'scroll', 'options': TimelineOptions}.

To keep things simple for the developer, we could still handle a simple string type (and construct using the default timeline options for that type).